src/app/token-option-field-component-factories/spend-for-lab-data-token-option-field-component-factory.ts

```diff
@@ -40,7 +40,12 @@ export class SpendForLabDataTokenOptionFieldComponentFactory extends TokenOption
                 .appendBuilder(createQuizFieldComponentBuilder(this.canvasService, environmentInjector))
                 .appendBuilder(createStartTimeComponentBuilder(environmentInjector))
                 .appendBuilder(createEndTimeComponentBuilder(environmentInjector))
-                .appendBuilder(createNewDueTimeComponentBuilder(environmentInjector, 'New Due Time for Lab Data Quiz'))
+                .appendBuilder(
+                    createNewDueTimeComponentBuilder(
+                        environmentInjector,
+                        'Change Until Date/Time for Lab Data Canvas Quiz to'
```

Comment:
It is not matched with the label used for the assignment field in `spend-for-assignment-resubmission-token-option-field-component-factory.ts`.